changes

abaransy opened the pull request.

abaransy committed:
changes

philgresh commented on frontend/src/components/modal/modal.jsx:
```diff
@@ -0,0 +1,102 @@
+`
+const ModalHeader = styled.div`
+    position: relative; 
+    top: -22px; 
```
Don't know if the `top` property is going to work with `position: relative;`... consider using negative margin if you need to pull the div up.

philgresh commented on frontend/src/components/modal/modal.jsx:
```diff
@@ -0,0 +1,102 @@
+
+const Button = styled.button`
+    font-size: 20px; 
+    font-weight: 750;
```
Go with `font-weight: 700` instead

philgresh committed:
Minor updates

Change font-weight from 750 to 700. Changed tab size to see if Prettier is upset about it.

philgresh committed:
Update modal.jsx

philgresh reviewed (approved):
Made minor changes in addition to Aatef's submission
